### ISSUE TITLE
Small typo -> postgreSQL -> PostgreSQL

### ISSUE THREAD

**Issue by danielwestermann** [pgbackrest/pgbackrest#2462]
Hi,

just noticed a small typo in the help output of the restore command:

```
pgbackrest help restore | grep postgreSQL
  --pg-path                           postgreSQL data directory
pgbackrest version
pgBackRest 2.52.1
```

Regards
Daniel

**Comment by pgstef** [comment]
Still applicable on `main` apparently:

```bash
$ pgbackrest version
pgBackRest 2.54dev
$ pgbackrest help restore |grep pg-path
  --pg-path                           postgreSQL data directory
$ pgbackrest help backup |grep postgreSQL
  --pg-database                       postgreSQL database [default=postgres]
  --pg-host                           postgreSQL host for operating remotely
  --pg-host-ca-file                   postgreSQL host certificate authority file
  --pg-host-ca-path                   postgreSQL host certificate authority path
  --pg-host-cert-file                 postgreSQL host certificate file
  --pg-host-cmd                       postgreSQL host pgBackRest command
  --pg-host-key-file                  postgreSQL host key file
  --pg-host-port                      postgreSQL host port when pg-host is set
  --pg-host-type                      postgreSQL host protocol type
  --pg-host-user                      postgreSQL host logon user when pg-host
  --pg-path                           postgreSQL data directory
  --pg-port                           postgreSQL port [default=5432]
  --pg-socket-path                    postgreSQL unix socket path
  --pg-user                           postgreSQL database user
```

And given the [tests](https://github.com/pgbackrest/pgbackrest/blob/main/test/src/module/command/helpTest.c#L375), it sounds "expected". Probably something linked to [this code](https://github.com/pgbackrest/pgbackrest/blob/main/src/command/help/help.c#L505). It might not consider the xml placeholder `<postgres/>` as an acronym maybe. Not sure what's wrong nor or to solve it though :thinking:

**Comment by danielwestermann** [comment]
> Do you mean that "postgreSQL" starts with a lower-case letter? That's a transformation done by the help code. I think we skip it for things that are all-caps but this doesn't match that pattern. I suppose we could check for more than one cap.

Yes, exactly. Looks strange that way

**Comment by dwsteele** [comment]
> Yes, exactly. Looks strange that way

Agreed. Proper names should always be rendered correctly. Fixed in eb2f279a.

**Comment by danielwestermann** [comment]
> > Yes, exactly. Looks strange that way
> 
> Agreed. Proper names should always be rendered correctly. Fixed in [eb2f279](https://github.com/pgbackrest/pgbackrest/commit/eb2f279a2998d1ed893c5d8984590ab4997a5f32).

Thank you
